src/mrpro/operators/NonUniformFastFourierOp.py

```diff
@@ -2,11 +2,13 @@
 
 from collections.abc import Sequence
 from dataclasses import astuple
+from enum import Enum
 from itertools import product
 from typing import Literal
 
 import torch
-from torchkbnufft import KbNufft, KbNufftAdjoint
+from pytorch_finufft.functional import finufft_type1, finufft_type2
+from torchkbnufft import KbNufftAdjoint
```

Comment:
torchkbnufft dependency is removed, but here we import from torchkbnufft ?!

Reply:
naaa, that is work in progress i guess ;)